.github/dependabot.yml

```diff
@@ -6,7 +6,7 @@ updates:
     schedule:
       interval: daily
   - package-ecosystem: terraform
-    directory: /Terraform_modules/ec2_instance
+    directory: /Terraform_modules/modules/aws/rds_serverless_cluster
```

Comment:
Remove `Terraform_modules/`

Reply:
This needs fixing further down in the terraform workflow, in the "Update dependabot.yml" step of the dependabot job

Reply:
think I have sorted this now